libraries/rush-lib/src/cli/actions/DeployAction.ts

```diff
@@ -150,7 +150,11 @@ export class DeployAction extends BaseRushAction {
         this.rushConfiguration.defaultSubspace
       );
       transformPackageJson = pnpmfileConfiguration.transform.bind(pnpmfileConfiguration);
-      if (!scenarioConfiguration.json.omitPnpmWorkaroundLinks) {
+      if (this.rushConfiguration.subspacesFeatureEnabled) {
+        const rushConfigurationProject = this.rushConfiguration.getProjectByName(mainProjectName);
+        const subspace = rushConfigurationProject?.subspace;
+        if (subspace) pnpmInstallFolder = subspace.getSubspaceTempFolder();
```

Comment:
```suggestion
        if (subspace) {
          pnpmInstallFolder = subspace.getSubspaceTempFolder();
        }
```

I'm surprised the ESLint rules didn't require this preferred syntax. 🤔